### DISCUSSION TITLE
using post command in scripts

### DISCUSSION THREAD

**Discussion by AliRahimiakbar** [ipxe/ipxe#1336] [Q&A]
hi  
i want to use post command in my script but it fails with this error:
post: command not found
can you guys help me to add this command or should i replace that in my scripts?

**Comment by NiKiZe** [comment]
All errors in iPXE also contains an URL that almost always has more info and suggestions. If not it helps us to know exactly which message you are getting.

It also is easier if you post the actual script you are using.

Where did you get `post` from?
What you probably want is https://ipxe.org/cmd/params

Do also note the build section: https://ipxe.org/cmd/params#build_options

**Reply by AliRahimiakbar** [reply]
actually i have this line in my script:
post url?bootmac=${mac}
and the url of error is : https://ipxe.org/err/2e0220

my script is using chains
that chian to one url and in that url exist other script that have this line in it
post url?bootmac=${mac}

**Reply by NiKiZe** [reply]
> actually i have this line in my script: post url?bootmac=${mac} and the url of error is : https://ipxe.org/err/2e0220
> 
> my script is using chains that chian to one url and in that url exist other script that have this line in it post url?bootmac=${mac}

There is no `post` command, see the `params` command as explained above if you want HTTP POST.
